crates/lsp/src/beancount_data.rs

```diff
@@ -15,25 +18,27 @@ pub struct FlaggedEntry {
 
 #[derive(Clone, Debug)]
 pub struct BeancountData {
-    accounts: Vec<String>,
+    // Assumption: an account can only have a definition within a beancount file.
+    pub(crate) accounts_definitions: HashMap<String, Range>,
     narration: Vec<String>,
-    pub flagged_entries: Vec<FlaggedEntry>,
+    pub(crate) flagged_entries: Vec<FlaggedEntry>,
     tags: Vec<String>,
     links: Vec<String>,
+    pub(crate) commodities_definitions: HashMap<String, Range>,
 }
```

Comment:
This is currently inconsistent, we should probably agree on what we want to do in terms of visibility (and getters).